db/people.yml

```diff
@@ -2373,3 +2373,50 @@ htiwari:
       status: ms
   name: Hrishi Tiwari
   position: Former Masters Researcher
+kknox:
+  active: false
+  aka:
+    - Kevin Knox
+    - Knox, K.
+    - Knox, Kevin
+  avatar: https://www.gravatar.com/avatar/00000000000000000000000000000000?d=mp&f=y
+  bio: bio
+  education:
+    - begin_year: 2006
+      degree: PhD
+      institution: Columbia University
+      end_year: 2011
+    - begin_year: 2004
+      degree: MA in Physics
+      institution: Columbia University
+      end_year: 2006
+    - begin_year: 1997
+      degree: BA in Physics
+      institution: Cornell University
+      end_year: 2001
+  employment:
+    - begin_year: 2015
+      begin_month: 7
+      organization: J.P. Morgan
+      department: []
+      position: Vice President
+    - begin_year: 2011
+      begin_month: 7
+      end_month: 6
+      end_year: 2015
+      group: bg
+      status: postdoc
+      advisor: sbillinge
+      organization: Brookhaven National Laboratory
+      position: Post Doctoral Associate
+    - begin_year: 2006
+      begin_month: []
```

Comment:
this needs to be integer or string.  If we don't know it , just delete the entry I think (not sure if that is a required field or not but we will find out).  Same for the end_month